armi/reactor/tests/test_parameters.py

```diff
@@ -509,228 +523,80 @@ class MockSyncPC(parameters.ParameterCollection):
 def makeComp(name):
     c = composites.Composite(name)
     c.p = MockSyncPC()
+    for pd in c.p.paramDefs:
+        pd.assigned = parameters.NEVER
```

Comment:
Did these need to be added? It appears that the same thing is done in the `SynchronizationTests.setUp()` method, which appears to be the only place that `makeComp()` is used.

Maybe you meant to remove the similar lines from the `setUp()` method?

Reply:
Fair point.  This is just something I didn't cleanup while trying various versions of these tests.

I removed them from the `setUp()` method.